test/Table_Tests/src/In_Memory/Table_Spec.enso

```diff
@@ -148,7 +148,7 @@ add_specs suite_builder =
         group_builder.specify "should handle error scenarios gracefully" <|
             Table.new [["X", [1,2,3]], ["Y", [4]]] . should_fail_with Illegal_Argument
             Table.new [["X", [1]], ["X", [2]]] . should_fail_with Illegal_Argument
-            Table.new [["X", [1]], ["Y", [2], "Z"]] . should_fail_with Illegal_Argument
+            Table.new [["X", [1]], ["Y", [2], Value_Type.Integer, "Z"]] . should_fail_with Illegal_Argument
```

Comment:
Can we also add a positive test here like
```
t = Table.new [["X", [Nothing], Value_Type.Integer]]
t.at "X" . to_vector . should_equal [Nothing]
t.at "X" . value_type . should_equal Value_Type.Integer
```
that checks that even if column is empty, it has the correct type?

Ideally I'd also like to have a negative test
```
Test.expect_panic Type_Error <| Table.new [["Y", [2], "Z"]]
```

Reply:
Done